### DISCUSSION TITLE
docker compose failed in risingwave-demo/docker

### DISCUSSION THREAD

**Discussion by MZGuo111** [risingwavelabs/risingwave#5868] [Q&A] [answered]
Sorry to bother you because the demo repo has no disscussion part. I use docker compose to build the demo in risingwave-demo/docker directory, and when pulling the image it failed. But it works at rising-demo/ad-click project. Need your help, thanks!
the error message is :
```
 docker compose up -d
[+] Running 0/9
 ⠇ compactor-0 Pulling                                                                                             3.9s
 ⠇ compute-node-0 Pulling                                                                                          3.9s
 ⠇ meta-node-0 Pulling                                                                                             3.9s
 ⠇ frontend-node-0 Pulling                                                                                         3.9s
   ⠦ d19f32bd9e41 Pulling fs layer                                                                                 0.6s
   ⠦ 3e26b730e723 Pulling fs layer                                                                                 0.6s
   ⠦ c5683cf412a8 Pulling fs layer                                                                                 0.6s
   ⠦ 4d06c5651fc6 Waiting                                                                                          0.6s
   ⠦ 7e568f809767 Waiting                                                                                          0.6s
error pulling image configuration: download failed after attempts=1: unknown blob
```

**Comment by lmatz** [comment, marked as answer]
Hi @MZGuo111, could you try updating all `ghcr.io/risingwavelabs/risingwave:nightly-20220901` in the `docker` directory to the same as the version in other directories, i.e. `ghcr.io/risingwavelabs/risingwave:nightly-20220926`? 

Just created the discussion page in the demo repo, thanks for the feedback! It's also fine to discuss here.

**Reply by MZGuo111** [reply]
Thank for your relay！The problerm is solved.

**Reply by lmatz** [reply]
> Thank for your relay！The problerm is solved.

More than happy to help!
@MZGuo111 Also feel free to join our community Slack Channel! 
Questions can be answered more timely😃
